contracts/protocol/facets/DisputeResolverHandlerFacet.sol

```diff
@@ -103,6 +103,14 @@ contract DisputeResolverHandlerFacet is IBosonAccountEvents, ProtocolBase {
             _disputeResolverFees.length <= protocolLimits().maxFeesPerDisputeResolver,
             INVALID_AMOUNT_DISPUTE_RESOLVER_FEES
         );
+
+        // Escalation period must be greater than zero and less than or equal to the max allowed
+        require(
+            _disputeResolver.escalationResponsePeriod > 0 &&
+                _disputeResolver.escalationResponsePeriod <= protocolLimits().maxEscalationResponsePeriod,
```

Comment:
If it doesn't cause a stack too deep error, we should probably fetch the protocolLimits() storage slot to a local var and reuse it for these two requires.

Reply:
done